raft/src/main/java/org/apache/kafka/snapshot/RecordsSnapshotWriter.java

```diff
@@ -146,7 +147,7 @@ final public static class Builder {
         private Time time = Time.SYSTEM;
         private int maxBatchSize = 1024;
         private MemoryPool memoryPool = MemoryPool.NONE;
-        private short kraftVersion = 1;
+        private KRaftVersion kraftVersion = KRaftVersion.KRAFT_VERSION_0; // TODO
```

Comment:
TODO reminder. What it is missing to make 1 the default for the builder?

Reply:
I'll change it to 1 since it was that way previously.